components/Dialogs/tests/snapshot/MDCAlertControllerLocalizationTests.m

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
```

Comment:
Tip: Nit: Please import UIKit and CoreGraphics.

Reply:
Addressed in 9ed4cbe40464c1fa6c7f23b34948efd3d7a61fbc.